changelog/README.md

```diff
@@ -0,0 +1,53 @@
+# changelog
+
+Release notes are text files with three lines:
+
+1. An opening code block with the `release-note:<MODE>` type annotation.
+
+   For example:
+
+       ```release-note:bug
+
+   Valid modes are:
+
+    - `bug` - Any sort of non-security defect fix.
+    - `security` - Any fixes for security issues in previous versions.
+    - `change` - A change in the product that may require action or
```

Comment:
Update wording to reflect "breaking change"

Reply:
Whoops! Updated.